autoload/battery/backend/termux.vim

```diff
@@ -18,8 +18,11 @@ function! s:on_stdout(buffer, data) abort
 endfunction
 
 function! s:on_exit(backend, buffer, exitval) abort
+	let g:bat_status_list = ['NOT_CHARGING' , 'DISCHARGING']
   let content = join(a:buffer, '')
-  let a:backend.is_charging = json_decode(content).status !=# 'NOT_CHARGING'
+	if index(g:bat_status_list, json_decode(content).status) >= 0
+		let a:backend.is_charging = 0
+	endif
   let a:backend.value = json_decode(content).percentage + 0
```

Comment:
Ok. Rewrite it like the below. **Use TWO SPACES instead of TAB to indent**.

```vim
function! s:on_exit(backend, buffer, exitval) abort
  let obj = json_decode(join(a:buffer, '{}'))
  let status = get(obj, 'status', '')
  let a:backend.is_charging = !(status ==# 'NOT_CHARGING' || status ==# 'DISCHARGING')
  let a:backend.value = get(obj, 'percentage', 0) + 0
endfunction
```

I didn't check if the above really work so please confirm the behavior by yourselves.

Reply:
This work, i have push it, thank you